aries_vcx/src/indy/ledger/transactions.rs

```diff
@@ -568,6 +591,13 @@ pub async fn add_service(wallet_handle: WalletHandle, pool_handle: PoolHandle, d
     Ok(res)
 }
 
+pub async fn add_service_public(wallet_handle: WalletHandle, pool_handle: PoolHandle, did: &str, service: &EndpointService) -> VcxResult<String> {
```

Comment:
- let's rename this function to `write_endpoint`
- let's rename the original function `add_service` to `write_endpoint_legacy`